src/UKHO.ERPFacade.API/appsettings.json

```diff
@@ -1,52 +1,53 @@
 {
-    "Logging": {
-        "LogLevel": {
-            "Default": "Information"
-        },
-        "ApplicationInsights": {
-            "LogLevel": {
-                "Default": "Information"
-            }
-        }
-    },
-    "AllowedHosts": "*",
-    "KeyVaultSettings": {
-        "ServiceUri": ""
-    },
-    "EventHubLoggingConfiguration": {
-        "MinimumLoggingLevel": "Information",
-        "UkhoMinimumLoggingLevel": "Information",
-        "Environment": "",
-        "EntityPath": "",
-        "System": "ERP",
-        "Service": "ERP-Facade",
-        "NodeName": "Azure",
-        "ConnectionString": ""
-    },
-    "AzureADConfiguration": {
-        "MicrosoftOnlineLoginUrl": "https://login.microsoftonline.com/",
-        "TenantId": "",
-        "ClientId": ""
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information"
     },
     "ApplicationInsights": {
-        "ConnectionString": ""
-    },
-    "AzureStorageConfiguration": {
-        "ConnectionString": ""
-    },
-    "SapConfiguration": {
-        "BaseAddress": "",
-        "SapServiceOperation": "Z_ADDS_MAT_INFO",
-        "Username": "",
-        "Password": ""
-    },
-    "SharedKeyConfiguration": {
-        "Key": ""
-    },
-    "EnterpriseEventServiceConfiguration": {
-        "ServiceUrl": "",
-        "PublishEndpoint": "api/events",
-        "ClientId": "",
-        "PublisherScope": ".default"
+      "LogLevel": {
+        "Default": "Information"
+      }
     }
+  },
+  "AllowedHosts": "*",
+  "KeyVaultSettings": {
+    "ServiceUri": ""
+  },
+  "EventHubLoggingConfiguration": {
+    "MinimumLoggingLevel": "Information",
+    "UkhoMinimumLoggingLevel": "Information",
+    "Environment": "",
+    "EntityPath": "",
+    "System": "ERP",
+    "Service": "ERP-Facade",
+    "NodeName": "Azure",
+    "ConnectionString": ""
+  },
+  "AzureADConfiguration": {
+    "MicrosoftOnlineLoginUrl": "https://login.microsoftonline.com/",
+    "TenantId": "",
+    "ClientId": ""
+  },
+  "ApplicationInsights": {
+    "ConnectionString": ""
+  },
+  "AzureStorageConfiguration": {
+    "ConnectionString": ""
+  },
+  "SapConfiguration": {
+    "BaseAddress": "",
+    "SapServiceOperation": "Z_ADDS_MAT_INFO",
+    "Username": "",
+    "Password": ""
+  },
+  "SharedKeyConfiguration": {
+    "Key": ""
+  },
+  "EnterpriseEventServiceConfiguration": {
+    "ServiceUrl": "",
+    "PublishEndpoint": "api/events",
+    "ClientId": "",
+    "PublisherScope": ".default"
+  },
+  "ApplicationUri": "https://erp.ukho.gov.uk"
```

Comment:
Can we move this under EnterpriseEventServiceConfiguration and rename it as ErpPublishEventSource ?